app.py

```diff
@@ -55,6 +70,7 @@ def add_scores(value):
 
     # weirdness_score.subscribe(lambda value: count = count + 1)
     weirdness_score.subscribe(add_scores)
+    weirdness_score.subscribe(print)
```

Comment:
lets not add new `print` subscribers. this makes it only more noisy. please remove this